src/Cli/Microsoft.DotNet.Cli.Utils/MSBuildPropertyParser.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections.Generic;
```

Comment:
This file is missing the MIT license header. Probably should run Ctrl+K+D to fix up formatting :)

Reply:
Thanks for the spot - updated.